src/SortPeopleByDateOfBirth.java

```diff
@@ -0,0 +1,25 @@
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class SortPeopleByDateOfBirth {
```

Comment:
It would be nice to see a test for this class so we know its sorting as we expect :)